<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volatility GUI by Pental</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="scan">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="Command">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="init_scan">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Init Scan</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IMAGE SCAN</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="open_file_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Open</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="file_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>741</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Command_Result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>741</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="work">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="cmdscan_btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>cmdscan</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pstree_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pstrss</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="imageinfo_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>imageinfo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
